Suez water: simplify config flow (#35638)
home-assistant/home-assistant.github.io: source/_integrations/suez_water.markdown

ha_integration_type: integration
---

> The Suez Water integration fetches your water consumption from the French water provider [Tout Sur Mon Eau](https://www.toutsurmoneau.fr) website.

Reviewer: ```suggestion
The **Suez Water** {% term integration %} fetches your water consumption data from the French water provider [Tout Sur Mon Eau](https://www.toutsurmoneau.fr) website.
```

> Water usage yesterday **sensor** fetch yesterday consumption if available.

Reviewer: ```suggestion
The **Water usage yesterday** sensor shows yesterday's water consumption data if that data is available.
```

{% include integrations/config_flow.md %}

`counter_id` is the water counter id.
It should be found automatically during setup.
If not it can be found on your _Tout Sur Mon Eau_ [user account](https://www.toutsurmoneau.fr/mon-compte-en-ligne/historique-de-consommation-tr). Look in the source code of the page for something similar to `url: '/mon-compte-en-ligne/statMData' + '/123456789'`. The `counter_id` in this case is `123456789`.

> 

Reviewer: ```suggestion
{% configuration_basic %}
Counter ID:
  description: "The water counter ID. It should be found automatically during setup. If it was not found, the ID can be found on your _Tout Sur Mon Eau_ [user account](https://www.toutsurmoneau.fr/mon-compte-en-ligne/historique-de-consommation-tr). Look in the source code of the page for something similar to `url: '/mon-compte-en-ligne/statMData' + '/123456789'`. The `counter_id` in this case is `123456789`."
{% endconfiguration_basic %}

```
This applies the formatting for the configuration options block
